discovery_engine/test/codecs/json_codecs_test.dart

```diff
@@ -61,7 +61,7 @@ void main() {
       final message_1 = converter.convert(request_1) as Map;
 
       expect(message_1[kSenderKey], isA<SendPort>());
-      expect(message_1[kPayloadKey], equals({'type': 'feedRequested'}));
+      expect(message_1[kPayloadKey], equals({'runtimeType': 'feedRequested'}));
```

Comment:
there is an option to still use the key `type` if we specify it in `build.yaml` file like so:
```yaml
targets:
  $default:
    builders:
      freezed:
        options:
          union_key: type
```

But I guess `runtimeType` is a better name, so let's go with that.